framework/SwiftPlot/PlotDimensions.swift

```diff
@@ -6,7 +6,10 @@ public struct PlotDimensions{
     public var graphWidth  : Float = 0
     public var graphHeight : Float = 0
 
-    public init(frameWidth : Float = 1000, frameHeight : Float = 660, subWidth sW : Float = 1000, subHeight sH : Float = 660) {
+    public init(frameWidth : Float = 1000,
+                frameHeight : Float = 660,
+                subWidth sW : Float = 1000,
+                subHeight sH : Float = 660) {
```

Comment:
As a stylistic note, it's general convention to have no spaces before a colon in function parameters, and one space afterward. (`frameWidth: Float`). Google's Swift style guide (which you don't have to adhere to, but is one example) has a section on that here:

https://google.github.io/swift/#horizontal-whitespace